lib/dor/assembly/content_metadata.rb

```diff
@@ -30,12 +30,14 @@ def stub_content_metadata_exists?
         File.exist?(stub_cm_file_name)
       end
 
+      # rubocop:disable Naming/MemoizedInstanceVariableName
       def load_content_metadata
         # Loads content metadata XML into a Nokogiri document.
         raise "Content metadata file #{Settings.assembly.cm_file_name} not found for #{druid.id} in any of the root directories: #{@root_dir.join(',')}" unless content_metadata_exists?
 
-        @cm = Nokogiri.XML(File.open(cm_file_name)) { |conf| conf.default_xml.noblanks }
+        @cm ||= Nokogiri.XML(File.open(cm_file_name)) { |conf| conf.default_xml.noblanks }
```

Comment:
Can you say more about how this change fits in this PR? Is it an opportunistic performance bump?

Reply:
It allows us to make expectations on the values returned from this.  This ensures that every call on the instance returns the same values.